app/controllers/profile/ProtectAccount.scala

```diff
@@ -138,7 +138,7 @@ class ProtectAccount @Inject()(val thirdPartyDeveloperConnector: ThirdPartyDevel
       Future.successful(BadRequest(protectAccountRemovalAccessCodeView(form)))
     },
       form => {
-        mfaService.removeMfa(loggedIn.email, form.accessCode).map(r =>
+        mfaService.removeMfa(loggedIn.developer.userId, loggedIn.email, form.accessCode).map(r =>
```

Comment:
What happens if loggedIn or loggedIn.developer is null?

Reply:
It can't be